packaging/pom.xml

```diff
@@ -37,14 +37,14 @@
         <dependency>
             <groupId>org.apache.brooklyn</groupId>
             <artifactId>brooklyn-all</artifactId>
-            <version>${project.version}</version>
+            <version>0.9.0-SNAPSHOT</version>
```

Comment:
What's the reason for using fixed versions? Using properties makes it easier to support.
If can't do without it, then mark it with  <!-- BROOKLYN_VERSION --> tag as done above.